src/types/codeBlock.ts

```diff
@@ -0,0 +1,12 @@
+export type CodeBlockEventData = {
```

Comment:
Can you please rename this file to `code-block.ts` (prefer all lowercase, kebab-case filenames) and update the import in `types/index.ts`?